src/activities/categorization/Zone.qml

```diff
@@ -40,15 +40,13 @@ Flow {
         model: zoneModel
         Item {
             id: item
-            width: horizontalLayout ? middleZone.width * 0.32 : middleZone.width * 0.49
-            height: horizontalLayout ? categoryBackground.height * 0.2 : categoryBackground.height * 0.15
+            width: middleScreen.width*0.32
```

Comment:
why did you change the width and height and removed the condition for horizontalLayout?